README.md

```diff
@@ -175,14 +180,15 @@ By default each event is enhanced with the following metadata:
 
 ```js
 {
-  event_name,
-    user_agent,
-    google_analytics_client_id,
-    connection_type,
-    effective_connection_type,
-    url,
-    timestamp,
-    application;
+  signal_version;
+  event_name;
+  user_agent;
+  google_analytics_client_id;
+  connection_type;
+  effective_connection_type;
+  url;
+  timestamp;
+  application;
```

Comment:
Are these supposed to be `;` in code? Is this supposed to resemble a TypeScript type?